### ISSUE TITLE
PostgreSQL 17.02 or higher is required, but you have 17.2.0

### ISSUE THREAD

**Issue by benyue1978** [timescale/homebrew-tap#51]
```shell
brew install timescaledb libpq
```
I got this error.
```
==> Installing timescale/tap/timescaledb
Error: PostgreSQL 17.02 or higher is required, but you have 17.2.0
```

Something is wrong with the latest commit for the postgresql version check.
I have to use these steps to workaround
```shell
cd $(brew --prefix)/Library/Taps/timescale/homebrew-tap
git checkout a71fd8a318a66b1e9883fbd137d35ea70951b417 timescaledb.rb
brew install timescaledb libpq
```

**Comment by JiriBucek** [comment]
Same problem here. I am trying to upgrade to the latest version. There seems to be some problem with how the version number is parsed


```bash
 brew upgrade timescaledb libpq                                                                                                                                                                                                                                                                                                                                                                                                                                                     ✔  18:02:05
Warning: libpq 17.2 already installed
==> Upgrading 1 outdated package:
timescale/tap/timescaledb 2.17.1 -> 2.17.2
==> Fetching dependencies for timescale/tap/timescaledb: postgresql@17
==> Fetching postgresql@17
==> Downloading https://ghcr.io/v2/homebrew/core/postgresql/17/manifests/17.2
Already downloaded: /Users/butcha/Library/Caches/Homebrew/downloads/032ded11eab504174b5d4db6d07ec24a6c811e09d0f4e64854cad9f5c7bfe674--postgresql@17-17.2.bottle_manifest.json
==> Downloading https://ghcr.io/v2/homebrew/core/postgresql/17/blobs/sha256:e0efcbfe6f8bcfa1c63820af7a076f9d8b91b3c609c2e0f5dbfbf8b4df2d29f0
Already downloaded: /Users/butcha/Library/Caches/Homebrew/downloads/f741434e9fddc146750d1ff0671ebf679e30e9a7b3d91b3141fa31c74b9a3ae6--postgresql@17--17.2.arm64_sequoia.bottle.tar.gz
==> Fetching timescale/tap/timescaledb
==> Downloading https://github.com/timescale/timescaledb/archive/refs/tags/2.17.2.tar.gz
Already downloaded: /Users/butcha/Library/Caches/Homebrew/downloads/2ad2b5c6bb8894ca572dc5d84827955714c4ba436928d86e61c0476e9ceafc33--timescaledb-2.17.2.tar.gz
==> Upgrading timescale/tap/timescaledb
  2.17.1 -> 2.17.2
==> Installing dependencies for timescale/tap/timescaledb: postgresql@17
==> Installing timescale/tap/timescaledb dependency: postgresql@17
==> Downloading https://ghcr.io/v2/homebrew/core/postgresql/17/manifests/17.2
Already downloaded: /Users/butcha/Library/Caches/Homebrew/downloads/032ded11eab504174b5d4db6d07ec24a6c811e09d0f4e64854cad9f5c7bfe674--postgresql@17-17.2.bottle_manifest.json
==> Pouring postgresql@17--17.2.arm64_sequoia.bottle.tar.gz
🍺  /opt/homebrew/Cellar/postgresql@17/17.2: 3,813 files, 71.4MB
==> Installing timescale/tap/timescaledb
Error: PostgreSQL 17.02 or higher is required, but you have 17.2.0
Error: An exception occurred within a child process:
  SystemExit: exit
```

**Comment by dsmooot** [comment]
same...

**Comment by jonatas** [comment]
Fixed on #53.
